client/src/utils/KBar.tsx

```diff
@@ -63,9 +65,114 @@ export const KBar: React.FunctionComponent<PropsWithChildren> = ({ children }) =
         dispatch(fetchAllCharacters())
       },
     },
+    {
+      id: 'configuration',
+      name: 'Choose demo configuration options...',
+      keywords: 'configuration',
+      section: 'Configuration',
+    },
+    {
+      id: 'ledger',
+      name: 'Select ledger',
+      keywords: 'ledger',
+      section: '',
+      parent: 'configuration',
+    },
+    {
+      id: 'bcovrin',
+      name: 'BCovrin Test',
+      keywords: 'bcovrin',
+      section: '',
+      perform: () => {
+        alert('BCovrin Test ledger selected!')
+      },
+      parent: 'ledger',
+    },
+    {
+      id: 'issue-credential-protocol-version',
+      name: 'Select credential protocol version',
+      keywords: 'issue credential protocol version',
+      section: '',
+      parent: 'configuration',
+    },
+    {
+      id: 'issue-credential-protocol-version-1',
+      name: 'V1',
+      keywords: 'issue credential protocol version 1',
+      section: '',
+      perform: () => {
+        dispatch(setProtocolVersion('v1'))
+      },
+      parent: 'issue-credential-protocol-version',
+    },
+    {
+      id: 'issue-credential-protocol-version-2',
+      name: 'V2',
+      keywords: 'issue credential protocol version 2',
+      section: '',
+      perform: () => {
+        dispatch(setProtocolVersion('v2'))
+      },
+      parent: 'issue-credential-protocol-version',
+    },
+    {
+      id: 'present-proof-protocol-version',
+      name: 'Select proof protocol version',
+      keywords: 'present proof protocol version',
+      section: '',
+      parent: 'configuration',
+    },
+    {
+      id: 'present-proof-protocol-version-1',
+      name: 'v1',
+      keywords: 'present proof protocol version 1',
+      section: '',
+      perform: () => {
+        alert('Proof Protocol Version 1 selected')
```

Comment:
I think the credential version and the invitation types are the only things currently supported. We can remove the other options for now.

Reply:
On it